dxr/utils.py

```diff
@@ -295,7 +295,16 @@ def split_content_lines(unicode):
     resulting lines.
 
     """
-    return unicode.splitlines(True)
+    lines = unicode.splitlines(True)
+    # Vertical Tabs and Form Feeds are treated as end-of-lines by splitlines
```

Comment:
Thanks for doing that! Let's go with the `in (u"\v", u"\f", u"\x1c", ...)` version, and easy choice since it both does the right thing and is fastest.